pyproject.toml

```diff
@@ -31,25 +31,23 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
-pelican = "^3 || ^4"
+pelican = ">=4.5"
```

Comment:
What do you think the minimum Pelican version should be?

Reply:
(Without having tested it) you might be away with the first version of Pelican than introduced plugins, which looks to be v3.0. I don't think we rely on any features added by more recent versions.

But my inclination is to have a very light touch with project dependencies...perhaps too light.

Reply:
(Whatever you decide here, don't hold it up over this)

Reply:
I believe we've been using v4.5 in other plugins because that version is when namespace plugins were introduced. Since the current canonical install method is via Pip, v4.5 is indeed required to Pip-install Pelican plugins [the way we tell people to do in the docs](https://github.com/pelican-plugins/image-process?tab=readme-ov-file#installation)

Reply:
I like your logic! Thanks again to putting this together.